### ISSUE TITLE
feature request: cli pretty printing

### ISSUE THREAD

**Issue by yschimke** [eiiches/jackson-jq#10]
This is one of the most useful features of jq

```
$ oksocial https://api.twitter.com/1.1/statuses/user_timeline.json | jq . | head -3
[
  {
    "created_at": "Sun Apr 10 06:13:19 +0000 2016",
```

```
$ oksocial https://api.twitter.com/1.1/statuses/user_timeline.json | java -jar ~/Downloads/jackson-jq-cli-0.0.4.jar .
[{"created_at":"Sun Apr 10 06:13:19 +0000 2016","id":719045577518899200,"id_str":"719045577518899200"...
```

**Comment by eiiches** [comment]
Added -c and -r option as in the jq. Will be released as 0.0.5 this week. Thanks!

**Comment by eiiches** [comment]
Sorry for the delay. I've released jackson-jq 0.0.5 to the maven central.

Also, I forked your tap repo and updated a recipe to the new version.
https://github.com/eiiches/homebrew-jackson-jq

**Comment by yschimke** [comment]
nice, worked first time for me
